corehq/apps/app_manager/static/app_manager/js/form_designer.js

```diff
@@ -164,6 +164,7 @@ hqDefine("app_manager/js/form_designer.js", function() {
             editDetails.setUpdateCallbackFn(function (name) {
                 $('#formdesigner .fd-content-left .fd-head-text').text(name);
                 $('.variable-form_name').text(name);
+                document.title = name + ' - CommCare HQ';
```

Comment:
fair point. @biyeun and I offlined, she offered to take a crack at that in another PR.